docs/documentation.md

```diff
@@ -356,6 +372,17 @@ Returns an array of listing IDs of the given `nftType` and `nftID`.
 
 ---
 
+**fun `cleanupGhostListings()`**
+
+```cadence
+pub fun cleanupGhostListings(listingResourceID: UInt64)
+```
+cleanupGhostListings
+Allow anyone to cleanup ghost listings. Listings will become ghost listings
+if stored provider capability doesn't hold the NFT anymore.
```

Comment:
So I think the above wording would also not be correct as well. It should be re-written like this

```
Allow callers to clean up ghost listings for this seller. Listings which remain orphaned on marketplaces because the stored provider capability cannot acquire the NFT any more.
```